onnxruntime/core/providers/cuda/activation/activations.h

```diff
@@ -173,5 +173,15 @@ class ThresholdedRelu final : public UnaryElementwise {
   float alpha_;
 };
 
+template <typename T>
```

Comment:
contrib activation should be under: https://github.com/microsoft/onnxruntime/tree/master/onnxruntime/contrib_ops/cuda/activation .

Reply:
thanks for the reminder, yes I should move it in the right place.